ivre/tools/ipdata.py

```diff
@@ -92,21 +94,23 @@ def my_parse_args():
                 ivre.db.db.data.ensure_indexes()
     if args.download:
         ivre.geoiputils.download_all(verbose=not args.quiet)
+    dbtofeed = {}
+    if not args.dont_feed_ipdata_cols:
+        dbtofeed["feedipdata"] = [ ivre.db.db.passive ]
+        if args.update_nmap_db:
```

Comment:
The idea behind `--dont-feed-ipdata-cols` was that updating `passive` DB was the default behavior. Renaming it to `--no-update-passive-db`, to be consistent with your new option (and updating the help accordingly) would be OK to me. Thanks!